index.html

```diff
@@ -1,163 +1,274 @@
             <div>
-                <button type="button">See Project</button>
+              <fieldset>
+                <input
+                  type="text"
+                  placeholder="Yeremias"
+                  maxlength="30"
+                  required
+                />
+              </fieldset>
             </div>
-            
-        </div>
-        <div class="card">
-            <img class="card-image" src="./img/portfolio3.jpg" alt="webpage image">
-            <h2>Tonic</h2>
-            <div class="tags">
-                <p>CANOPY</p>
-                <img class="bullet" src="./img/Counter.png" alt="bullet">
-                <span>Back End Dev</span>
-                <img class="bullet" src="./img/Counter.png" alt="bullet">
-                <span>2015</span>
+            <div>
+              <fieldset>
+                <input type="email" placeholder="natajayanj14@gmail.com" required />
+              </fieldset>
             </div>
-            <p class="description">
-                A daily selection of privately personalized reads. No accounts or 
-                sign-ups required.
-            </p>
-            <ul class="technologies">
-                <li>html</li>
-                <li>css</li>
-                <li>javaScript</li>
-            </ul>
             <div>
-                <button type="button">See Project</button>
+              <fieldset>
+                <textarea
+                  name="comment"
+                  placeholder="Write your message here"
+                  maxlength="500"
+                  required
+                ></textarea>
+              </fieldset>
             </div>
-        </div>
-    </section>
-    <section class="about">
-        <div class="about-content">
-            <div class="container">
-                <h2>About myself</h2>
-                <p>Hello I'm a software developer! I can help you build a product,  
-                feature or website. Look through some of my work and experience!
-                If you like what you see and have a project you need coded, don't hesitate 
-                to contact me. 
-                </p>
-                <span>LET'S CONNECT</span>
-                <ul class="social-media">
-                    <li><a href="#"><img src="./img/twitter.png" alt="twitter icon"></a></li>
-                    <li><a href="https://www.linkedin.com/in/jiachun-lily-xiang-1b197213a/"><img src="./img/linkedin.png" alt="linkedin icon"></a></li>
-                    <li><a href="#"><img src="./img/M.png" alt="M icon"></a></li>
-                    <li><a href="https://github.com/mpa-mxiang"><img src="./img/git.png" alt="github icon"></a></li>
-                    <li><a href="#"><img src="./img/hand.png" alt="hand icon"></a></li>
-                </ul>
-                <a><button type="button">Get my resume</button></a>  
+          </div>
+            <button class="button" type="submit">Get in touch</button>
```

Comment:
- [ ] You did well making the `button` adjustment suggested by the previous reviewer, however, the hover and press functionality is not implemented. it was stated by the preview reviewer, but I believe you missed it, but that's fine. 😃  please add the function based on the figma design to give a proposed user a better experience when working with your web app. 👍 

# Yours remain the same on all clicks:
![image](https://user-images.githubusercontent.com/50990925/213353094-3050a9b3-9066-4c45-9a54-bc05e2271380.png)


# Figma:
![image](https://user-images.githubusercontent.com/50990925/213353012-506e67e4-5e84-491e-9101-1b9d9ad5455d.png)